acceptance-tests/storage_migration_test.go

```diff
@@ -0,0 +1,89 @@
+package acceptance_test
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"csbbrokerpakgcp/acceptance-tests/helpers/apps"
+	"csbbrokerpakgcp/acceptance-tests/helpers/brokers"
+	"csbbrokerpakgcp/acceptance-tests/helpers/gcloud"
+	"csbbrokerpakgcp/acceptance-tests/helpers/random"
+	"csbbrokerpakgcp/acceptance-tests/helpers/services"
+)
+
+var _ = Describe("Storage Migration", Label("storage-migration"), func() {
+	It("can migrate instance from the previous legacy broker to the CSB", func() {
+
+		By("creating an Storage bucket using the legacy broker")
+
+		sourceBucketName := random.Name(random.WithPrefix("old"), random.WithMaxLength(15))
+		legacyServiceInstance := services.CreateInstance(
+			"google-storage",
+			"multiregional",
+			services.WithBroker(&brokers.Broker{Name: legacyBrokerName}),
```

Comment:
It's nice to use `WithBrokerName()` here, but that update might not have been merged yet.

Reply:
Great. Changed.